cmd/cli/uninstall_mesh.go

```diff
@@ -199,6 +182,63 @@ func (d *uninstallMeshCmd) run() error {
 			fmt.Fprintf(d.out, "OSM namespace CANNOT be deleted in a managed environment\n")
 		}
 	}
+
+	err := d.deleteClusterResources()
+	return err
+}
+
+func (d *uninstallMeshCmd) findFlagSpecifiedMesh(meshInfoList []meshInfo) (bool, bool) {
```

Comment:
nit, since the method doesn't "know" that the fields are specified via flag or not, it shouldn't "leak" that info through its name.

`findMesh` seems like a descriptive name for this method

Reply:
`findMesh` is already an existing function, I changed the function name to `findSpecifiedMesh` and the function signature to take in the `flagMeshSpecified` bool.